### DISCUSSION TITLE
Stream audio from HAOS to seperate raspberry pi

### DISCUSSION THREAD

**Discussion by WazzA1987** [music-assistant/support#3051] [Q&A] [answered]
Will MPD help me?

I have a raspberry pi running home assistant. I have another networked rapsberry pi with sone speakers plugged in to the audio jack. I would like to stream audio from HAOS to the 'remote' raspberry pi's speakers.

Thanks 😀

**Comment by marcelveldt** [comment, marked as answer]
I would install either squeezelite (slimproto) or snapcast on that remote pi.

**Comment by WazzA1987** [comment]
> I would install either squeezelite (slimproto) or snapcast on that remote pi. 

Thanks For that,
Got it working exactly what I needed. Thanks again :)
